src/backend/libs/efiling-account-client/src/test/java/ca/bc/gov/open/jag/efilingaccountclient/config/AutoConfigurationTest.java

```diff
@@ -1,22 +1,32 @@
 package ca.bc.gov.open.jag.efilingaccountclient.config;
 
-import ca.bc.gov.open.jag.efilingaccountclient.EfilingAccountService;
+import com.sun.xml.internal.ws.wsdl.parser.InaccessibleWSDLException;
+import org.junit.jupiter.api.Assertions;
 import org.junit.jupiter.api.DisplayName;
 import org.junit.jupiter.api.Test;
 import org.springframework.boot.test.context.runner.ApplicationContextRunner;
 
-import static org.assertj.core.api.Assertions.assertThat;
-
 @DisplayName("AutoConfiguration Test")
 public class AutoConfigurationTest {
     ApplicationContextRunner context = new ApplicationContextRunner()
             .withUserConfiguration(AutoConfiguration.class);
 
+
+    private AutoConfiguration sut;
+
+
     @Test
     @DisplayName("CASE1: Test that beans are created")
     public void testBeansAreGenerated() {
-        context.run(it -> {
-            assertThat(it).hasSingleBean(EfilingAccountService.class);
+
+
+        CsoAccountProperties properties = new CsoAccountProperties();
+        properties.setFilingAccountSoapUri("http://localhost");
```

Comment:
No port?

Reply:
It's a test that's meant to not be able to connect I think.

Reply:
Can't read apparently :|